.github/workflows/enclave-cc-cicd.yaml

```diff
@@ -0,0 +1,59 @@
+name: CI/CD | enclave-cc e2e tests
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator enclave-cc e2e tests
+    runs-on: ubuntu-22.04
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: network=host
+
+      - name: Build and push to local registry
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: localhost:5000/cc-operator:latest
+
+      - name: Setup kind cluster
+        run: |
+          kind create cluster -n coco-sgx --config tests/e2e/enclave-cc-kind-config.yaml
```

Comment:
is `kind` installed in the environment?

Reply:
Yes, it's.  I've copied this from the https://github.com/confidential-containers/operator/blob/main/.github/workflows/e2e.yaml, which is up and running with this as it is. :-)

Reply:
yes `kind` is pre-installed in Github virtual envs.

Reply:
You also could pass: 
 * `--wait duration` so it wait the control node be ready.
 * --image "kindest/node:v1.24.0" to use k8s 1.24

Reply:
> Yes, it's. I've copied this from the https://github.com/confidential-containers/operator/blob/main/.github/workflows/e2e.yaml, which is up and running with this as it is. :-)

oh, just realized this PR is a copy of the an existing e2e workflow.

Reply:
Yep, basically that, but with a few change to actually use the payload generated for the Ci, and to run it every day.

I've propagated the changes you suggested to that workflow as well.